doc/manual/plugins_local/saveimage.rst

```diff
@@ -0,0 +1,29 @@
+.. _local-plugin-saveimage:
```

Comment:
This documentation needs to move to `plugins_global/saveimage.rst`. This file needs to be deleted.